modin/backends/base/query_compiler.py

```diff
@@ -2571,8 +2592,8 @@ def view(self, index=None, columns=None):
         BaseQueryCompiler
             New masked QueryCompiler.
         """
-        index = [] if index is None else index
-        columns = [] if columns is None else columns
+        index = slice(None) if index is None else index
```

Comment:
This is a bug in a `BaseQueryCompiler.view` regarding to the incorrect default values (implicitly causes some problems in a Groupby.__getitem__ for a base backend). Ideally, this should be a separate PR fixing this, but maybe we can keep it here since the fix is just about fixing the typo?

Reply:
Let's leave this here.

Reply:
I would prefer to move this to a new PR to track the change at a finer granularity.